File: cakk-domain/src/main/java/com/cakk/domain/config/JpaConfig.java
package com.cakk.domain.config;

import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@Configuration
@EnableJpaAuditing
@EnableTransactionManagement
@EntityScan("com.cakk.domain.entity.**")
@EnableJpaRepositories("com.cakk.domain.repository.**")
public class JpaConfig {
}

File: cakk-infrastructure/persistence-mysql/src/main/java/com/cakk/infrastructure/persistence/config/JpaConfig.kt
package com.cakk.infrastructure.persistence.config

import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaAuditing
import org.springframework.data.jpa.repository.config.EnableJpaRepositories
import org.springframework.transaction.annotation.EnableTransactionManagement

@Configuration
@EnableJpaAuditing
@EnableTransactionManagement
@EntityScan("com.cakk.infrastructure.persistence.entity.**")
@EnableJpaRepositories("com.cakk.infrastructure.persistence.repository.**")
class JpaConfig
